Mutagen.Bethesda.Analyzers.Skyrim/Record/ConditionAnalyzer.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Mutagen.Bethesda.Analyzers.SDK.Analyzers;
+using Mutagen.Bethesda.Analyzers.SDK.Results;
+using Mutagen.Bethesda.Analyzers.SDK.Topics;
+using Mutagen.Bethesda.Skyrim;
+namespace Mutagen.Bethesda.Analyzers.Skyrim.Record;
+
+public class ConditionAnalyzer : IIsolatedRecordAnalyzer<ISkyrimMajorRecordGetter>
+{
+    private static readonly TopicDefinition<string?> InvalidConditionReference = MutagenTopicBuilder.DevelopmentTopic(
+            "Condition Runs on Null Reference",
+            Severity.Error)
+        .WithFormatting<string?>("Condition {0} runs on reference, but reference is null");
```

Comment:
I think most topics of type error should have some notes on what the known problems they cause in-game are.

Normally this would go in the Discussion topic on github.  But while it's just a `DevelopmentTopic`, we need to keep the notes somewhere.  If it was initiated by an issue, we'd keep them there.. but for now maybe add comments?

Otherwise, when we eventually "upgrade" it off `DevelopmentTopic`, I'm not gonna know the driving original details of why it was an error